--- build.xml	2012-03-01 01:56:24.000000000 +0100
+++ build.xml-gil	2012-05-16 12:33:02.759366191 +0200
@@ -58,6 +58,7 @@
     <property name="lib.jsp12" location="lib/jsp-api-1.2.jar"/>
     <property name="lib.jsp20" location="lib/jsp-api-2.0.jar"/>
     <property name="lib.jsp21" location="lib/jsp-api-2.1.jar"/>
+    <property name="lib.el" location="lib/el-api.jar"/>
     
     <property name="lib.jython20" location="lib/jython-2.0.jar"/>
     <property name="lib.jython22" location="lib/jython-2.2.1.jar"/>
@@ -86,11 +87,14 @@
     <path id="compile.classpath">
         <path refid="compile.nomultiversion.classpath"/>
         <pathelement path="${lib.jsp21}"/>
+        <pathelement path="${lib.el}"/>
     </path>
 
     <path id="compile.jsp1.classpath">
         <path refid="compile.nomultiversion.classpath"/>
         <pathelement path="${lib.jsp12}"/>
+        <pathelement path="${lib.jsp21}"/>
+        <pathelement path="${lib.el}"/>
     </path>
 
     <path id="compile.jsp2.classpath">
@@ -119,6 +123,8 @@
         <pathelement path="${lib.jdom}"/>
         <pathelement path="${lib.servlet}"/>
         <pathelement path="${lib.jsp20}"/>
+        <pathelement path="${lib.jsp21}"/>
+        <pathelement path="${lib.el}"/>
     </path>
 
     <path id="compile.classpath.javadoc">
@@ -139,6 +145,8 @@
         <pathelement path="${src.dir}"/>
         <pathelement path="${java.class.path}"/>
         <pathelement path="${lib.javarebel-sdk}"/>
+        <pathelement path="${lib.jsp21}"/>
+        <pathelement path="${lib.el}"/>
     </path>
 
     <path id="compile.classpath.examples">
@@ -297,6 +305,7 @@
 
     <target name="fetchlibs" unless="all.libs.present" description="fetch build dependencies and put them in lib directory">
         <mkdir dir="lib"/>
+<!--
         <echo>Fetching libraries from server.</echo>
         <get src="${libs.location}/README.txt" dest="lib/README.txt" usetimestamp="true"/>
         <get src="${libs.location}/ant.jar" dest="lib/ant.jar" usetimestamp="true"/>
@@ -317,7 +326,7 @@
         <get src="${libs.location}/log4j.jar" dest="lib/log4j.jar" usetimestamp="true"/>
         <get src="${libs.location}/2.3/logkit.jar" dest="lib/logkit.jar" usetimestamp="true"/>
         <get src="${libs.location}/slf4j-api.jar" dest="lib/slf4j-api.jar" usetimestamp="true"/>
-        <get src="${libs.location}/slf4j-jdk14.jar" dest="lib/slf4j-jdk14.jar" usetimestamp="true"/> <!-- Needed only for testing SLF4J: -->
+        <get src="${libs.location}/slf4j-jdk14.jar" dest="lib/slf4j-jdk14.jar" usetimestamp="true"/> < Needed only for testing SLF4J:>
         <get src="${libs.location}/commons-logging.jar" dest="lib/commons-logging.jar" usetimestamp="true"/>
         <get src="${libs.location}/2.3/rt122.jar" dest="lib/rt122.jar" usetimestamp="true"/>
         <get src="${libs.location}/saxpath.jar" dest="lib/saxpath.jar" usetimestamp="true"/>
@@ -326,7 +335,7 @@
         <get src="${libs.location}/jsp-api-1.2.jar" dest="lib/jsp-api-1.2.jar" usetimestamp="true"/>
         <get src="${libs.location}/jsp-api-2.0.jar" dest="lib/jsp-api-2.0.jar" usetimestamp="true"/>
         <get src="${libs.location}/jsp-api-2.1.jar" dest="lib/jsp-api-2.1.jar" usetimestamp="true"/>
-        <get src="${libs.location}/xalan.jar" dest="lib/xalan.jar" usetimestamp="true"/>
+        <get src="${libs.location}/xalan.jar" dest="lib/xalan.jar" usetimestamp="true"/-->
     </target>
 
     <target name="init">
@@ -547,7 +556,7 @@
         <mkdir dir="${build.classes.dir}"/>
         <javac deprecation="off"
                 destdir="${build.classes.dir}" debug="on" optimize="off"
-                bootclasspath="lib/rt122.jar" source="1.2" target="1.2"
+                bootclasspath="lib/rt122.jar" source="6" target="6"
                 includeantruntime="false">
             <src path="${src.dir}"/>
             <patternset refid="freemarker.compile"/>
@@ -561,13 +570,13 @@
         <antcall target="compile-commons-logging"/>
         <antcall target="compile-jdk14"/>
         <antcall target="compile-jdk15"/>
-        <antcall target="compile-jsp1"/>
+        <!--antcall target="compile-jsp1"/-->
         <antcall target="compile-jsp2"/>
         <antcall target="compile-jython20"/>
         <antcall target="compile-jython22"/>
         <antcall target="compile-jython25"/>
         <antcall target="compile-dom4j"/>
-        <antcall target="compile-jdom"/>
+        <!--antcall target="compile-jdom"/-->
         <antcall target="compile-xalan"/>
         <antcall target="compile-jaxen"/>
         <antcall target="compile-rhino"/>
@@ -580,7 +589,7 @@
 
     <target name="compile-log4j" if="log4j.available" depends="init">
         <javac destdir="${build.classes.dir}"
-                bootclasspath="lib/rt122.jar" target="1.2" source="1.2"
+                bootclasspath="lib/rt122.jar" target="6" source="6"
                 deprecation="off" debug="on" optimize="off"
                 includeantruntime="false">
             <src path="${src.dir}"/>
@@ -591,7 +600,7 @@
 
     <target name="compile-logkit" if="logkit.available" depends="init">
         <javac destdir="${build.classes.dir}"
-                bootclasspath="lib/rt122.jar" target="1.2" source="1.2"
+                bootclasspath="lib/rt122.jar" target="6" source="6"
                 deprecation="off" debug="on" optimize="off"
                 includeantruntime="false">
             <src path="${src.dir}"/>
@@ -602,7 +611,7 @@
 
     <target name="compile-slf4j" if="slf4j.available" depends="init">
         <javac destdir="${build.classes.dir}"
-                bootclasspath="lib/rt122.jar" target="1.2" source="1.2"
+                bootclasspath="lib/rt122.jar" target="6" source="6"
                 deprecation="off" debug="on" optimize="off"
                 includeantruntime="false">
             <src path="${src.dir}"/>
@@ -613,7 +622,7 @@
 
     <target name="compile-commons-logging" if="commons-logging.available" depends="init">
         <javac destdir="${build.classes.dir}"
-                bootclasspath="lib/rt122.jar" target="1.2" source="1.2"
+                bootclasspath="lib/rt122.jar" target="6" source="6"
                 deprecation="off" debug="on" optimize="off"
                 includeantruntime="false">
             <src path="${src.dir}"/>
@@ -631,7 +631,7 @@
 
     <target name="compile-jdk14" if="jdk14.available" depends="init">
         <javac destdir="${build.classes.dir}"
-                target="1.4" source="1.4"
+                target="6" source="6"
                 deprecation="off" debug="on" optimize="off"
                 includeantruntime="false">
             <src path="${src.dir}"/>
@@ -642,7 +642,7 @@
 
     <target name="compile-jdk15" if="jdk15.available" depends="init">
         <javac destdir="${build.classes.dir}"
-                target="1.5" source="1.5"
+                target="6" source="6"
                 deprecation="off" debug="on" optimize="off"
                 includeantruntime="false">
             <src path="${src.dir}"/>
